healthScore/views.py

```diff
@@ -1045,3 +1045,18 @@ def view_healthworkers_user_record(request):
         )
 
     return homepage(request)
+
+
+def get_patients(request):
+    patients = list(User.objects.filter(is_patient=True).values())
+    return JsonResponse({'patients': patients})
+
```

Comment:
Could you please add test cases for these functions?

Reply:
will do